### ISSUE TITLE
Indicators for scenario start and end dates

### ISSUE THREAD

**Issue by NXXR** [DLR-SC/ESID#296]
As scenarios could have different start dates, indicating the start and end dates for the scenarios could be useful to the user.
<details><summary>Mockup (click to expand)</summary>

![grafik](https://github.com/DLR-SC/ESID/assets/25636783/da22b1ef-9bd2-43e1-a903-2dfbbb7dd89f)

</details> 
Vertical indicators in the line chart could be used to show the range of the scenario.
Additionally, the selected scenario could have labels with the date.

**Comment by JonasGilg** [comment]
I think this is already heading for too much clutter. There are some more issues, for example what are the rates being compared to in the cards?

**Comment by NXXR** [comment]
> I think this is already heading for too much clutter. There are some more issues, for example what are the rates being compared to in the cards?

I agree with the additional clutter. Maybe we could hide it bz default and make it a toggle in the settings if there are multiple scenarios with different start dates? and then only show when it's actually useful.

Regarding the rates this would be a different issue anyways.  
In general each rate should be relative to the start date of its scenario?  
Or should the rate be relative to the latest available case data?

maybe @lenarothenhaeusler has some idea or insight from the pilot study on what info would be useful, or what the selected date values should be compared against.

If there are multiple options we'd need to decide on a default behavior and push the other options into a data card settings menu.

**Comment by lenarothenhaeusler** [comment]
If i get it right you asked about the percentages in the scenario box and if it makes sense here to compare it to the start date or relative to another date? I don't have feedback from the pilot study regarding the comparison of the rates. So i suppose the health authorities were happy with how its implemented at the moment. In my opinion it makes totally sense to compare it to the scenario start date. I guess that in most cases it would be the current date (so the latest case data) anyway and i guess one want to see the percentage increase or decrease from the start date. Also if its relative to another date it gets confusing with the dates i suppose...

**Comment by NXXR** [comment]
> Just an idea from my side: regarding the vertical indicators and too much clutter, would it be an option to just show the start and end dates of a specific scenario with vertical indicators when you move on the box with the mouse?? If you remove the mouse it doesn't show the vertical lines anymore then.

You mean showing the indicators when the user hovers over a data card?
It would be a little tricky but in general possible. Although, I would look into the possibility to only add the indicators after the user kept hovering on top of the card for a specific time (0.5~1.5 seconds) to avoid adding and removing the indicators too often.  
Alternatively, we could show the indicators for a few seconds after selecting or clicking on the data card and then remove them.

**Comment by lenarothenhaeusler** [comment]
> You mean showing the indicators when the user hovers over a data card?

Yes, thats what i meant^^
